pyoos/collectors/collector.py

```diff
@@ -47,12 +48,15 @@ def get_bbox(self):
         return self._bbox
 
     def set_bbox(self, bbox):
-        if bbox is not None:
-            if isinstance(bbox, list) or isinstance(bbox, tuple):
-                bbox = tuple(bbox)
-            else:
-                raise ValueError("Not a recognized bbox. \
-                                  Must be in format: (minx, miny, maxx, maxy)")
+        bbox = tuple(bbox)
+        dx, dy = -1, -1
+        if len(bbox) == 4:
+            dx = bbox[2] - bbox[0]
+            dy = bbox[3] - bbox[1]
+        if not dx > 0 and not dy > 0:
```

Comment:
If I got this right we avoid users shooting themselves in the foot with a wrong `bbox` and provide a better info a bad bbox is provided.

Reply:
should this be an or? And just a suggestion, I find the logical NOTs harder to read than if you were to say 

```
if dx <= 0 and dy <= 0:
```

Reply:
`or` is better b/c it should fail if any of those evaluates and I agree about the logical NOT, thanks!